Makefile

```diff
@@ -127,18 +127,35 @@ build: airflow-build
 
 build-dev: airflow-dev-build
 
+clean:
+	$(DOCKER_COMPOSE) down -v
+
+airflow-initdb:
+	$(DOCKER_COMPOSE) run --rm  webserver initdb
+
+
+end2end-test:
+	$(MAKE) clean
+	$(MAKE) airflow-initdb
+	$(DOCKER_COMPOSE) run --rm  test-client
+	$(MAKE) clean
+
 ci-build-dev:
 	$(MAKE) DOCKER_COMPOSE="$(DOCKER_COMPOSE_CI)" build-dev
 
 ci-test-exclude-e2e: build-dev
-	$(DOCKER_COMPOSE_CI) run --rm peerscout-dags-dev ./run_test.sh
+	$(MAKE) DOCKER_COMPOSE="$(DOCKER_COMPOSE_CI)" \
+	 run --rm peerscout-dags-dev ./run_test.sh
```

Comment:
Same as the other issues. In this case we could delegate to `test`, in which case we could probably delete `run_test.sh`.
Alternatively leave it as it was.